packages/scripts/config/.prettierrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	useTabs: true,
+	tabWidth: 2,
```

Comment:
Should this be defined? Some devs like 4 spaces, some 2, even 8.

Leaving this as a developer preference for them to set in their Editor/IDE is in line with how WordPress Core uses `.editorconfig` by not defining a _tab width_

Reply:
Thanks for the explanation 👍🏼